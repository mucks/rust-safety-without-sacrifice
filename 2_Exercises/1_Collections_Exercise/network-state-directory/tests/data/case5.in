# Mixed scenario
MEMBER ForestCity 1 Alice Citizen
MEMBER ForestCity 2 Alex Citizen
MEMBER Prospera 3 Priya Founder
MEMBER Zuitzerland 4 Zack Validator
MOVE Zuitzerland ForestCity 4
PREFIX ForestCity Al
COUNT ForestCity
STATS
