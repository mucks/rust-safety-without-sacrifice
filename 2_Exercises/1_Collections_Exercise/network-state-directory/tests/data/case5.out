Moved 4 from Zuitzerland to ForestCity: true
PREFIX ForestCity Al:
 - Alice (1)
 - Alex (2)
COUNT ForestCity: 3
STATS citizens=2 validators=1 founders=1
